ReactNativeClient/lib/components/screens/config.js

```diff
@@ -118,6 +118,7 @@ class ConfigScreenComponent extends BaseScreenComponent {
 					const files = await shim.fsDriver().readDirStats(Setting.value('resourceDir'));
 
 					for (const file of files) {
+						if (file.path == 'pluginAssets') continue;
```

Comment:
A better fix would be to recursively copy all files and folders. I think there's an option on readDirStats for that

Reply:
Okay I'll check them out.

Reply:
I tried passing recursive option `{ recursive: true }` but now it shows different error:
![Screenshot_1587212616](https://user-images.githubusercontent.com/15380913/79637492-b84b1b80-819d-11ea-9340-729955f7781b.png)

@laurent22  If you have any other better way of fixing the issue then please let me know.

Reply:
The path on your screenshot looks suspicious - why is `/data/user/0/net.cozic.joplin/files/` appears twice? Looks like a bug somewhere

Reply:
@kowalskidev  are you able to debug the origin of the alert

Reply:
I'm working on it